From b561a922829c9ff9b31aca7a01a79c1ce475db37 Mon Sep 17 00:00:00 2001
From: Marek Vasut <marex@denx.de>
Date: Mon, 4 Jan 2016 13:20:51 +0100
Subject: [PATCH 21/29] fpga: fpga-bridge: Fix the l3 regs compat name

Use correct name for the L3 regs DT node, which is altr,l3-nic.
The previous altr,l3regs is not used.

Signed-off-by: Marek Vasut <marex@denx.de>
---
 drivers/fpga/altera-hps2fpga.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/fpga/altera-hps2fpga.c b/drivers/fpga/altera-hps2fpga.c
index c15df47..a2f0bd6 100644
--- a/drivers/fpga/altera-hps2fpga.c
+++ b/drivers/fpga/altera-hps2fpga.c
@@ -141,7 +141,7 @@ static int alt_fpga_bridge_probe(struct platform_device *pdev)
 		return PTR_ERR(priv->bridge_reset);
 	}
 
-	priv->l3reg = syscon_regmap_lookup_by_compatible("altr,l3regs");
+	priv->l3reg = syscon_regmap_lookup_by_compatible("altr,l3-nic");
 	if (IS_ERR(priv->l3reg)) {
 		dev_err(dev, "regmap for altr,l3regs lookup failed\n");
 		return PTR_ERR(priv->l3reg);
-- 
2.10.2

